### DISCUSSION TITLE
How to turn on spa/pool heat from dashPanel?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#455] [Q&A] [answered]
I can see the status of the heater and Solar (Heat Pump) from dashPanel, but I can't figure out how to tun on and off or set temp.

Does dashPanel support that?

**Comment by davthomaspilot** [comment, marked as answer]
Figured it out.  Sorry, don't know how to delete post.

**Reply by rstrouse** [reply]
No worries it was hidden in plain sight
